### DISCUSSION TITLE
Render and Validate only one field from useFieldArray

### DISCUSSION THREAD

**Discussion by inomn** [react-hook-form/react-hook-form#9469] [Q&A]
I have a giant form under one `control` and parts of the form are rendered conditionally.
Example: user wants to order 5 items. With some general info, I'm creating a `useFieldArray` with a set of fields. To provide this additional data, user clicks on a button and Modal window appears.

What I want to do is:
1. Validate only current field `from` `useFieldArray` `fields` in open Modal by pressing a button
2. Update the valid\invalid state of the whole form
3. Form should have an invalid state at the beginning, but since fields are rendered conditionally in Modal, form state changes only when Fields actually rendered

Maybe I'm doing it completely wrong? The main issue for me is to render only one field at time and keep all validation working. Thanks.

**Comment by bluebill1049** [comment]
can you share a simple codesandbox?
